content/markdown/research-distribution-center/historical-imagery-archive.md

```diff
@@ -12,34 +13,37 @@ abstract: Our archive consists of over 1,000,000 frames with dates as far back a
-    </ul>
+<div class="row">
+  <div class="col-md-6 acquiring-agency">
+      <h3>United States Geological Survey</h3>
+      <a href="https://tnris-org-static.s3.amazonaws.com/images/usgs_full.jpg" data-toggle="lightbox" data-gallery="example-gallery" data-title="United States Geological Survey (USGS)">
+          <img class="thumbnail pull-left" src="https://tnris-org-static.s3.amazonaws.com/images/usgs_historical_th.jpg">
+      </a>
+      <ul class="list-clean">
+          <li><strong>Media Type:</strong> Black and White, Color, and Color Infrared prints</li>
+          <li><strong>General Scales:</strong> 1:3000 – 1:125000</li>
+          <li>TNRIS houses extensive United States Geological Survey (USGS) collections encompassing the state of Texas.  Dates of collections span from the early 1930s to the late 2000s.</li>
+          <li><strong>About the USGS:</strong><br> The Aerial Photography Single Frame Records collection is a large and diverse group of imagery acquired by federal organizations from 1937 to present.  The USGS, one of the participating federal organizations, houses it’s historical imagery at the USGS Long Term Archive (LTA) at the National Center for Earth Resources Observations and Sciences (EROS) in Sioux Falls, SD.</li>
+      </ul>
+  </div>
+  <div class="col-md-6 acquiring-agency">
+      <h3>Texas General Land Office</h3>
+      <a href="https://tnris-org-static.s3.amazonaws.com/images/glo_full.jpg" data-toggle="lightbox" data-gallery="example-gallery" data-title="Texas General Land Office (GLO)">
+          <img class="thumbnail pull-left" src="https://tnris-org-static.s3.amazonaws.com/images/glo_historical_th.jpg">
+      </a>
+      <ul class="list-clean">
+          <li><strong>Media Type:</strong> Black and White, Color Infrared, Natural Color prints</li>
+          <li><strong>General Scales:</strong> 1:3000</li>
+          <li>TNRIS houses collections from the Texas General Land Office (GLO) from the early 1970s to the 1980s.  The majority of the collections archived at TNRIS are of the Texas coast, neighboring cities and communities, and of mining sites owned by the GLO in west Texas.</li>
+          <li><strong>About the GLO:</strong><br>The GLO was the first Texas public agency established in 1836 responsible for managing lands and mineral rights owned by the state. The agency has historical imagery collections for Texas from the 1930s to present.</li>
+      </ul>
+  </div>
 </div>
-<div class="acquiring-agency">
+<div class="row">
+<div class="col-md-6 acquiring-agency">
     <h3>Army Map Service</h3>
     <a href="https://tnris-org-static.s3.amazonaws.com/images/ams_full.jpg" data-toggle="lightbox" data-gallery="example-gallery" data-title="Army Map Service (AMS)">
-        <img class="thumbnail pull-left" src="https://tnris-org-static.s3.amazonaws.com/images/ams_th.jpg">
+        <img class="thumbnail pull-left" src="https://tnris-org-static.s3.amazonaws.com/images/armycorp_historical_th.jpg">
```

Comment:
just wanted to point this out in case you missed it, but the army map service image here is the only logo/image that has color compated to the other ones on the page. not sure if thats what you intended or not but if you're cool with it than I am.